src/jit/compiler.h

```diff
@@ -3099,6 +3101,18 @@ class Compiler
     bool compSupportsHWIntrinsic(InstructionSet isa);
     bool isScalarISA(InstructionSet isa);
 #endif // _TARGET_XARCH_
+#ifdef _TARGET_ARM64_
+    NamedIntrinsic lookupHWIntrinsic(const char* className, const char* methodName);
+    GenTree* impHWIntrinsic(NamedIntrinsic        intrinsic,
```

Comment:
Same for this one, identical other than the name. Would be good to share if we can.

Reply:
I had initially renamed the X86 version, but I had too many merge conflicts.  So I dropped it.